localization/marker_tracking/include/marker_tracking/marker_detector.h

```diff
@@ -19,6 +19,12 @@
 #ifndef MARKER_TRACKING_MARKER_DETECTOR_H_
 #define MARKER_TRACKING_MARKER_DETECTOR_H_
 
+#ifndef CV__ENABLE_C_API_CTORS
+#define CV__ENABLE_C_API_CTORS
+#endif
+#include <opencv2/imgproc.hpp>
```

Comment:
usually better not to put this in the header unless we really need to

Reply:
That function is in opencv2/imgproc.hpp, but it will only be defined if CV__ENABLE_C_API_CTORS is enabled

Reply:
It doesn't seem like RGB is called in the header though? Is there a reason we can't put it in the C++ function?

Reply:
Another solution would be to modify the alvar repo instead

Reply:
Huh, I wonder why alvar compiles then... I guess this is fine, maybe make a comment why / when this is needed.